devel/glaze/Portfile

```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem              1.0
+PortGroup               github 1.0
+PortGroup               cmake 1.1
+
+github.setup            stephenberry glaze 4.3.0 v
+github.tarball_from     archive
+revision                0
+categories              devel
+platforms               any
+supported_archs         noarch
+license                 MIT
+maintainers             {@sikmir disroot.org:sikmir} openmaintainer
+description             Extremely fast, in memory, JSON and interface library for modern C++
+long_description        {*}${description}
+
+checksums               rmd160  61c6920a3bc4879695d6955e913729c5799118c2 \
+                        sha256  72cd174ad8839bc526a679a5a8ebf4bb18f3cb850a34fcfd3e89970f83c4d384 \
+                        size    546532
+
+compiler.cxx_standard   2023
```

Comment:
I doubt that this does anything; is this actually defined in "macports-base"?

Reply:
also, if the port is "noarch" it implies nothing gets compiled so why would this be needed at all?

Reply:
But what about tests?

Reply:
> I doubt that this does anything; is this actually defined in "macports-base"?

I'm not sure, but it's used:
```
sysutils/caps-log/Portfile:34:compiler.cxx_standard   2023
textproc/vectorscan/Portfile:58:compiler.cxx_standard   2023
```